sub-logs/arafat.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/yasar-arafath/registergood",
+
+  "owner": {
+    "username": "yasar-arafath",
+    "email": "arafat8426@gmail.com"
+  },
+
+  "target": {
+    "record-type": {
+       "name": "www", 
```

Comment:
Record type is not the same as the file name you're requesting.

Reply:
Changed the record type